src/charm.py

```diff
@@ -137,12 +125,11 @@ def _on_start(self, event: ops.charm.StartEvent) -> None:
             logger.debug("opening tcp port")
             self._open_port_tcp(self._port)
         except subprocess.CalledProcessError:
-            self.unit.status = BlockedStatus(
-                "failed to open TCP port for MongoDB")
+            self.unit.status = BlockedStatus("failed to open TCP port for MongoDB")
```

Comment:
```suggestion
            self.unit.status = BlockedStatus("failed to open TCP port for MongoDB")
```

Not sure if this warrants a `BlockedStatus` - might just be worth logging with `logger.error`?